delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1049,41 +882,16 @@ def disableCloudFrontDistribution(distributionID, _Etag, configFile, stage) {
 	}
 }
 
-/**
- * Delete CloudFront Distribution configuration
- * @param distributionID
- * @param _Etag
- * @return
- */
-def deleteCloudFrontDistribution(distributionID, _Etag) {
-	// @TODO: Send an Event with distributionID, _Etag as inputs. A separate job will pick up the details and schedule for deletion
-	// @TODO: Clean up code below after that
-	/*
-	def deleteOutput
-	try {
-		deleteOutput = sh(
-			script: "aws cloudfront delete-distribution \
-						--id $distributionID \
-						--if-match $_Etag",
-			returnStdout: true
-		)
-		echo "deleteOutput...$deleteOutput"
-		return deleteOutput
-	} catch(ex) {
-		error "deleteCloudFrontDistribution. "+e.getMessage()
-	}
-	*/
-}
-
-def loadServerlessConfig(String runtime, String configPackBBURL) {
+def loadServerlessConfig(String runtime) {
 
-	echo "loadServerlessConfig::" + runtime + "::" + configPackBBURL
+	def configPackURL = scmModule.getCoreRepoCloneUrl("serverless-config-pack")
```

Comment:
Should this be pulled from from configuration?

Reply:
I want to consolidate all our platform specific stuff into one repo (jazz-build-module -> maybe rename this too a more meaningful name then). That will be for another story though. What I can do though is push this to a class member variable and reference it that way. Does that make sense?